app/assets/javascripts/ckeditor/plugins/cortex_media_insert/plugin.js

```diff
@@ -1,19 +1,19 @@
-(function (global) { 
+(function (global) {
   'use strict';
 
   global.CKEDITOR.plugins.add('cortex_media_insert', {
     icons: 'media',
     init: function (editor) {
       editor.addCommand('insertMedia', {
         exec: function (editor) {
-          window.MODALS.featured.open();
+          window.MODALS.wysiwyg.open();
```

Comment:
@arelia of the tree popups featured-tile-wysiwyg this deals with images/assets to be inserted into the wysiwyg. It was a point I made in a past PR so it's good to see the fix.